the-content-guide/readme.md

```diff
@@ -7,14 +7,100 @@ The target audience are communication professionals. See it as a comprehensive g
+
+## Structure & readability
+
+Readers are looking for answers and want to find them in the easiest, fastest way. A well structured text gives guidance and leads the reader towards its conclusion in a gentle, organic fashion. A clear text consists of:
+
+* The title
+* An intro
+* Headers
+* **Bold** and *italic*  
+* Short paragraphs
+* Clear quotes
+
+A strong and remarkable *title* is important to attract your reader. A good title should activate, interest and excite. Often enough with newspaper articles—also the digital ones— the title is the key component in attracting the reader.
+
+If you have lured your reader with a fantastic title, he/she often decides whether to keep on reading or not based on the first few sentences. This brings us to the intro of the text and the importance hereof.
+
+> If the title and image are the bait, the intro is the line that has to be strong enough to pull the reader in.
+
+The *intro* is officiously the first paragraph, and also supports a well chosen title. With this beginning the reader is either triggered and will stay, or not. Readers will scan the intro and decide in just a matter of seconds whether they want to keep on reading. The intro can consist of a promise that you make. Remember that readers are tend to find answers to their 'problems'. They were lured in with the title as a hook and now they really want to know of the content is worth there precious time.
+
+**Bold** en *italic* words make sure that highlighted concepts are cleared up. *Short paragraphs* are there to make the text understandable for the reader. *Clear quotes*, then, invite the reader to keep on reading.
+
+## Visitor behavior
+
+With the ever growing content, the bad and the good, content developers should know how readers behave on the web and how they
+
+According to Jacob Nielsen of the [Nielsen Norman Group](http://www.nngroup.com/articles/how-long-do-users-stay-on-web-pages/), a visitor spends an average of less than a minute on a webpage. The visitor usually only has time to read a quarter of the text presented. If the content has a clear message or benefit the chances it will be read are heightened. Quality content aims to give the reader new insights about a specific topic.
+
+## Search Engine Optimisation (SEO) or keyword research, here we go:
+
+Search engine optimisation is a way to (sort of) ensure that people find your website trough search engines like google, bing, google and google.
+
+We believe that the quality, originality and creative components of made content always stands above SEO strategy. However, we do acknowledge SEO as an important strategy when it comes to the visibility of the content. It would be a shame when a written masterpiece floats on the web and isn't picked up by an audience.
```

Comment:
"It would be a shame when a written masterpiece floats on the web and isn't picked up by an audience."--> "It would be a shame for a written masterpiece to float on the web and not be picked up by an audience." Because you used "would", the idea is conceptual. When is concrete.